ui-components/src/overlays/Modal.scss

```diff
@@ -34,6 +34,11 @@
   &:last-of-type {
     @apply rounded-b;
   }
+
+  &.is-scrollable {
+    height: 80vh;
```

Comment:
![Screen Shot 2022-05-31 at 5 27 26 PM](https://user-images.githubusercontent.com/65367387/171300956-c5345063-ef1b-4695-9b46-a2b146a75fc0.png)
This style can result in the footer being cut off at small screen sizes. Some sort of calc based on the view height I think will prevent this, something like `max-height: calc(100vh - 200px)` - let me know what you think!